node_helper.js

```diff
@@ -559,6 +570,25 @@ module.exports = NodeHelper.create({
     return [];
   },
 
+  /* NASA APIs documented under https://api.nasa.gov/.
+    This accesses the NASA Image and Video Library. 
+    Currently, it only loads the thumbnails, which in most cases have good enough quality.
+    For NASA API usage without an API key, there are hourly limits of about 1,000 requests. 
+    */
+  processNasaData: function (config, data) {
+    let images = [];
+    for (const image of data.collection.items) {
+      if (image?.links && image?.links?.length > 0) {
+        images.push({
+          url: image?.links[0]?.href,
+          caption: image?.data[0]?.description?.substring(0, 250)
```

Comment:
Does a 250 character caption still look good?  Perhaps it should use `title`, and have an option to use the description instead.  I see that some items with long descriptions also include a `description_508` field, which is an abbreviated version of the description, and could be used if present.